cli/src/pcluster/resources/login_node/user_data.sh

```diff
@@ -215,5 +216,7 @@ else
   timeout ${Timeout} /tmp/bootstrap.sh || error_exit
 fi
 
+cfn-signal --stack ${AWS::StackName} --resource ${AutoScalingGroupName} --region ${AWS::Region} --exit-code 0
```

Comment:
`cfn-signal` is not the tool we should use here. We should use [aws autoscaling group complete-lifecycle-action](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/autoscaling/complete-lifecycle-action.html), instead.

`cfn-signal` is the CloudFormation tool that allows you to send a positive/negative signal to a CloudFormation stack for a given CloudFormation resource. It's meant to tell CloudFormation the outcome of the creation/update for a given resource.

This is not want we want to do here, even if it is similar to it.

Our goal here is to notify the service Autoscaling Group (not CloudFormation) that the given AutoScalingGroup (not CloudFormation resource) can complete its lifecycle hook.

Reply:
The solution should be something like this:

```
aws autoscaling complete-lifecycle-action --lifecycle-action-result CONTINUE --instance-id $INSTANCE_ID --lifecycle-hook-name "${lifecycleHookName}" --auto-scaling-group-name "${autoScalingGroupName}"  --region ${AWS::Region}
```

Reply:
Done! Thank you for correcting.